packages/grant-explorer/src/features/round/ViewCartPage/RoundInCart.tsx

```diff
@@ -41,6 +119,15 @@ export function RoundInCart(
             removeProjectFromCart={props.handleRemoveProjectFromCart}
             project={project}
             index={key}
+            matchingEstimateUSD={
+              matchingEstimates
+                ?.flat()
+                .find(
+                  (est) =>
+                    getAddress(est.recipient ?? zeroAddress) ===
+                    getAddress(project.recipient ?? zeroAddress)
+                )?.differenceInUSD
```

Comment:
Moving calculations out of JSX might help with clarity, it's a very crude form of separating logic from presentation (logic above this line, presentation below this line) but still a form.